test/client.test.ts

```diff
@@ -294,35 +303,35 @@ describe('Tests CapacitorClient', () => {
     });
   });
 */
-    /* TODO: Fix SDKInfo
-  describe('event data enhancement', () => {
-    test('event contains sdk default information', async () => {
-      const mockedSend = jest.fn<PromiseLike<void>, [Envelope]>().mockResolvedValue(undefined);
-      const mockedTransport = (): Transport => ({
-        send: mockedSend,
-        flush: jest.fn().mockResolvedValue(true),
-      });
-      const client = new CapacitorClient({
-        ...DEFAULT_OPTIONS,
-        dsn: EXAMPLE_DSN,
-        transport: mockedTransport,
-      });
-
-      client.captureEvent({ message: 'test event' });
-
-      expect(mockedSend).toBeCalled();
-      const actualEvent: Event | undefined = <Event>(
-        mockedSend.mock.calls[0][firstArg][envelopeItems][0][envelopeItemPayload]
-      );
-      expect(actualEvent?.sdk?.packages).toEqual([
-        {
-          name: SDK_PACKAGE_NAME,
-          version: SDK_VERSION,
-        },
-      ]);
+  /* TODO: Fix SDKInfo
```

Comment:
L: From what I see this is unrelated to the PR, looks like only the indentation changed. 

What needs to be fixed with SDKInfo?

Reply:
Yeah only the indentation changed.

I made a PR only for SDKInfo separately just to avoid fixing everything on a single PR.

It's basically missing the default export from the SDK on the integration.